packages/ai-bot/lib/matrix.ts

```diff
@@ -126,7 +126,13 @@ export const toMatrixMessageCommandContent = (
   eventToUpdate: string | undefined,
 ): IContent | undefined => {
   let { arguments: payload } = functionCall;
-  const body = payload['description'] || 'Issuing command';
+  // Take the description if it exists (old format)
+  // Otherwise take the description from the attributes (new format, responses are cards)
+  // If neither exists, use a default
+  const body =
+    payload['description'] ||
+    payload['attributes']?.description ||
```

Comment:
@IanCal I don't think we should ever use this one. What prompted you to add it?

Reply:
if I'm picturing the layering correctly this is what we currently do for the patch command (it takes a description as an input which is what we show in the message text) but I'd be happy to remove it.